docs/source/user_guide/tutorials/getting_started.rst

```diff
@@ -253,484 +133,518 @@ For example, if you ran the ``easylink run`` command on your laptop, the impleme
 and so on.
 
-Let's run this same pipeline with the ``slurm`` computing environment. `Slurm <https://slurm.schedmd.com/overview.html>`_ is an open-source job scheduler and 
-cluster management system which EasyLink can interface with to schedule and run the steps of a pipeline using the resources of a computing cluster. This means that instead of 
-running all pipeline steps in your local computing environment, each step can be run with the additional resources of a separate compute node.
-
-To run the pipeline using slurm, we will pass :download:`environment_slurm.yaml <environment_slurm.yaml>` 
-to the ``--computing-environment`` command line parameter. Download the file to the directory you will run ``easylink`` 
-in -- I downloaded it to the root ``easylink`` directory. The YAML looks like this:
-
-.. code-block:: yaml
-
-   computing_environment: slurm
-   container_engine: singularity
-   slurm:
-      account: proj_simscience
-      partition: all.q
-   implementation_resources:
-      memory: 1  # GB
-      cpus: 1
-      time_limit: 1  # hours
+Input data
+----------
+The ``--input-data`` (``-i``) argument to ``easylink run`` accepts a YAML file specifying a list 
+of paths to files or directories containing input data to be used by the pipeline. 
+We passed ``input_data_demo.yaml``, the contents of which are shown below::
 
-The ``account`` and ``partition`` parameters are specific to your Slurm cluster configuration - you may need 
-to ask your system administrator for these. The parameters shown above would work for someone on the Simulation 
-Science team at IHME. For more information see the `Slurm docs <https://slurm.schedmd.com/overview.html>`_.
+  input_file_ssa: 2020/input_file_ssa.parquet
+  input_file_w2: 2020/input_file_w2.parquet
+  known_clusters: known_clusters.parquet
 
-The ``implementation_resources`` parameter specifies the compute resources which will be reserved by the Slurm 
-system for the implementation container for each step, including a ``time_limit`` for the job's execution.
+Here we have defined the locations of the three input files we will use: the 2020 versions of the 
+``Social Security Administration`` and ``W2 & 1099`` datasets, and an empty ``known_clusters`` file, since no
+clusters are known to us before running this pipeline. 
 
 .. note::
-   When using the ``slurm`` environment, you may have to wait for the computing resources your jobs need to become 
-   available on the cluster. The wait time will depend on how busy your cluster is with jobs submitted by other users. 
+    To meet the input specifications for :ref:`datasets` defined by the pipeline schema (see the next section),
+    the ``SSA`` and ``W2`` datasets, after being generated by pseudopeople, were modified
+    to add the required ``Record ID`` column. ``SSA`` death records were also removed, 
+    leaving only ``creation`` type records.
```

Comment:
Were the death records also removed to meet the specifications for datasets, or was this just done to simplify things?